### DISCUSSION TITLE
data inserted vice versa (nocodb and with python mysqli script)  is not inserted on other side

### DISCUSSION THREAD

**Discussion by extremety1989** [nocodb/nocodb#1863] [General]
windows 10
nodejs v16.14.2
mysqli3

i am using nocodb and vs-code to display mysqli db ;  so when i insert data into db with python script i can see them inside mysqli db on vs-code but not in nocodb (only the columns are displayed) ,next i tried to insert data with nocodb i can see it on nocodb but not in
vs-code

**Comment by o1lab** [comment]
Did you try refreshing nocodb or vs-code ? Moving this to discussion.

**Comment by extremety1989** [comment]
@o1lab yes and connection too

**Reply by o1lab** [reply]
This looks like a basic usecase. Would need more detailed steps on whats happening (may be a small video will help)

**Reply by extremety1989** [reply]
i found error, it because sqli does not support default values..

**Comment by extremety1989** [comment]
@o1lab 
p :: TypeError: String expected
    at C:\Users\TOSHIBA\nocodb\node_modules\knex\lib\dialects\sqlite3\index.js:65:18
    at new Promise (<anonymous>)
    at Client_SQLite3.acquireRawConnection (C:\Users\TOSHIBA\nocodb\node_modules\knex\lib\dialects\sqlite3\index.js:64:12)

schema.py
``
from sqlalchemy import Boolean, Column, ForeignKey, Integer, String

from sqlalchemy.orm import relationship

from database import Base

class User(Base):

    __tablename__ = "users"

    id = Column(Integer, primary_key=True, index=True)

    email = Column(String, unique=True, index=True)

    password = Column(String)

    is_active = Column(Boolean)

``

**Comment by extremety1989** [comment]
@o1lab does nocodb support int and bool types ?
